codegen.mts

```diff
@@ -6,6 +7,62 @@ const config: CodegenConfig = {
   emitLegacyCommonJSImports: true,
   generates: {
     // API
+    './packages/services/api/src': {
+      hooks: {
+        afterOneFileWrite: ['prettier --write'],
+      },
+      ...defineConfig({
+        typeDefsFilePath: false,
+        resolverGeneration: 'minimal',
+        resolverMainFileMode: 'modules',
+        resolverTypesPath: './__generated__/types.next.ts',
+        blacklistedModules: [
+          'activity',
+          'admin',
+          'alerts',
+          'auth',
+          'billing',
+          'cdn',
+          'collection',
+          'feedback',
+          'integrations',
+          'lab',
+          'oidc-integrations',
+          'operations',
+          'organization',
+          'policy',
+          'project',
+          'rate-limit',
+          'schema',
+          'shared',
+          'support',
+          'target',
+          'token',
+        ],
```

Comment:
I will go through this list one or a few modules at a time. 
This PR starts with `usage-estimation` to test it out.